docs/resources/backup_config.md

```diff
@@ -0,0 +1,117 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cockroach_backup_config Resource - terraform-provider-cockroach"
+subcategory: ""
+description: |-
+  The backup configuration for a cluster.
+  Each cluster has a backup configuration that determines if backups are enabled,
+  how frequently they are taken and how often they are retained for.  This
+  resource allows configuring those settings.  It's important to note that the
+  existence or lack of this resource doesn't not indicate whether a backup
+  configuration exists or not. Rather, if Terraform is managing it or not.
+  As a result of this, removal of the resource from the Terraform configuration
+  will leave the backup configuration in the state it was before the removal.
+  retention_days Edit Restrictions
+  retention_days can only be set once.  Further changes require opening a support
+  ticket.  For this reason, managing the value via terraform can require special
+  care. Here are some useful tips:
```

Comment:
```suggestion
  `retention_days` can only be set once. It is necessary to open a support ticket to modify the settings again. For this reason, consider the following when managing the value in Terraform:
```

Reply:
Since this only applies to this one setting, I've taken the suggestion but updated it to:

modify the setting again.